corehq/apps/api/resources/v0_4.py

```diff
@@ -37,6 +37,8 @@
 MOCK_XFORM_ES = None
 MOCK_CASE_ES = None
 
+xform_doc_types = doc_types()
```

Comment:
Using the constant naming scheme is probably a good idea (`XFORM_DOC_TYPES`) since this is a global. That way it's a big red flag to anyone writing or reading code that mutates its value. An alternate approach would be to do this assignment around line 90, before `def wrapper(doc)`.